src/components/dac_dataset_table/DACDatasetTable.js

```diff
@@ -0,0 +1,173 @@
+import { useState, useEffect } from 'react';
+import { h } from 'react-hyperscript-helpers';
+import { Styles } from '../../libs/theme';
+import { Storage } from '../../libs/storage';
+import PaginationBar from '../PaginationBar';
+import SimpleTable from '../SimpleTable';
+import cellData from './DACDatasetTableCellData';
+import {isNil} from 'lodash/fp';
+import {goToPage as updatePage, recalculateVisibleTable} from '../../libs/utils';
+import {useCallback} from 'react';
+import './dac_dataset_table.css';
+
+export const styles = {
+  baseStyle: {
+    fontFamily: 'Montserrat',
+    fontSize: '1.6rem',
+    fontWeight: 400,
+    display: 'flex',
+    padding: '1rem 2%',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    whiteSpace: 'pre-wrap',
+    backgroundColor: 'white',
+    border: '1px solid #DEDEDE',
+    borderRadius: '4px',
+    margin: '0.5% 0'
+  },
+  columnStyle: Object.assign({}, Styles.TABLE.HEADER_ROW, {
+    justifyContent: 'space-between',
+    color: '#7B7B7B',
+    fontFamily: 'Montserrat',
+    fontSize: '1.2rem',
+    fontWeight: 'bold',
+    letterSpacing: '0.2px',
+    textTransform: 'uppercase',
+    backgroundColor: 'B8CDD3',
+    border: 'none'
+  }),
+  cellWidths: {
+    duosId: '15%',
+    dataSubmitter: '15%',
+    datasetName: '25%',
+    dataCustodian: '15%',
+    dataUse: '15%',
+    status: '15%'
+  },
+  color: {
+    dataUseGroup: '#000000',
+    votes: '#000000',
+    numberOfDatasets: '#000000',
+    datasets: '#000000',
+  },
+  fontSize: {
+    dataUseGroup: '1.4rem',
+    votes: '1.4rem',
+    numberOfDatasets: '1.4rem',
+    datasets: '1.4rem',
+  },
+};
+
+export const DACDatasetTableColumnOptions = {
+  DUOS_ID: 'duosId',
+  DATA_SUBMITTER: 'dataSubmitter',
+  DATASET_NAME: 'datasetName',
+  DATA_CUSTODIAN: 'dataCustodian',
+  DATA_USE: 'dataUse',
+  STATUS: 'status',
+};
+
+const columnHeaderConfig = {
+  duosId: {label: 'DUOS ID', cellStyle: {width: styles.cellWidths.duosId}, cellDataFn: cellData.duosIdCellData, sortable: true},
+  dataSubmitter: {label: 'Data Submitter', cellStyle: {width: styles.cellWidths.dataSubmitter}, cellDataFn: cellData.dataSubmitterCellData, sortable: true},
+  datasetName: {label: 'Dataset Name', cellStyle: {width: styles.cellWidths.datasetName}, cellDataFn: cellData.datasetNameCellData, sortable: true},
+  dataCustodian: {label: 'Data Custodian', cellStyle: {width: styles.cellWidths.dataCustodian}, cellDataFn: cellData.dataCustodianCellData, sortable: true},
+  dataUse: {label: 'Data Use', cellStyle: {width: styles.cellWidths.dataUse}, cellDataFn: cellData.dataUseCellData, sortable: false},
+  status: {label: 'Status', cellStyle: {width: styles.cellWidths.status}, cellDataFn: cellData.statusCellData, sortable: false},
```

Comment:
Why turn off sorting for any of these?  Funny enough, this was the first column I wanted to sort on.  :-)

Reply:
As @connorlbark might have mentioned, I think we need a better term to describe this column, or even consider status (accepted, pending) separate from actions.  I haven't looked to see what Yes/No is all about, but it isn't clear from the UI what I'd be acting on.